### PR TITLE
[comparison-branch] Remove all the JS

### PR DESCRIPTION
**Pull request by wlonk**
**DO NOT MERGE**

This is just to aid comprehensive review of the JSX.

### CODE REVIEW

File: src/js/components/header/index.js [hunk 1]
```diff
@@ -1,70 +0,0 @@
...
-            slds-p-right_large
-            slds-text-link_reset
-            slds-align-middle"
-        >
-          Products
-        </Link>
-        {user ? <Logout user={user} doLogout={doLogout} /> : <Login />}
-      </>
-    }
-    variant="objectHome"
```
[the remark this answers is not in the document]
Comment (jgerigmeyer) [reply src/js/components/header/index.js]: `objectHome` is the variant which has content on the right side of the header. The specific classes aren't necessary in this case, but come along with using the correct variant.

File: src/js/components/header/index.js [hunk 2]
```diff
@@ -1,70 +0,0 @@
...
-}: {
-  user: User,
-  doLogout: typeof logout,
-}) => (
-  <PageHeader
-    className="global-header
-      slds-p-horizontal_x-large
-      slds-p-vertical_medium"
-    title={
-      <Link
```
[the remark this answers is not in the document]
Comment (jgerigmeyer) [reply src/js/components/header/index.js]: I think it's not a high priority, but I'm all for it if you have a quick suggestion!

File: src/js/components/plans/stepsTable.js [hunk 3]
```diff
@@ -1,191 +0,0 @@
...
-  preflight: ?PreflightType,
-}) => (
-  // DataTable uses step IDs internally to construct unique keys,
-  // and they must be strings (not integers)
-  <article className="slds-card slds-scrollable_x">
-    <DataTable
-      items={plan.steps.map(step => ({ ...step, id: step.id.toString() }))}
-      id="plan-steps-table"
-    >
-      <DataTableColumn
```
[the remark this answers is not in the document]
Comment (jgerigmeyer) [reply src/js/components/plans/stepsTable.js]: Hm, I think you're right... but unfortunately that's hard-coded into the SLDS `accordion` spec. So, I think we'd need to find a new design, or make our own modified accordion that doesn't fit the styleguide. Maybe a question for someone more familiar with SLDS?

**Comment by jgerigmeyer** [comment]
@stacyk Were there other comments that got lots? Or just these three?

**Comment by wlonk** [comment]
@stacyk Let me know when we can close this branch, or just close it yourself if you prefer, when you're done with it.

File: src/js/components/plans/stepsTable.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (stacyk) [reply src/js/components/plans/stepsTable.js]: I also see a <p> tag inside the <span> tag

    <h3>
        <button>
            <svg></svg>
            <span>
                <p>Quick step</p>
            </span>
        </button>
    </h3>

**Pull request closed without merging**